src/lib.rs

```diff
@@ -81,7 +81,7 @@ impl FileTime {
     /// let ft_i64 = FileTime::now().filetime();
     /// ```
     pub fn filetime(&self) -> i64 {
-        (self.secs * Self::HUNDREDS_OF_NANOSECONDS) + self.nsecs
+        (self.secs * Self::HUNDREDS_OF_NANOSECONDS) + self.nsecs.checked_div(100).unwrap_or(0)
```

Comment:
`checked_div()` only returns `None` [if the argument is 0](https://doc.rust-lang.org/std/primitive.u32.html#method.checked_div), which is never the case here. Shouldn't it be just a normal flooring integer division with `nsecs / 100`? In my understanding, time values shouldn't be rounded, but only floored - just like with time values in a video player.

(Same below.)

Reply:
thx, you are absolutely right. I was thinking backwards, in regards to a divison by null... which of course is not the case here, as the division is done by a literal value of 100. Corrected